Tests/Behat/FlowContextTrait.php

```diff
@@ -272,11 +254,6 @@ protected function resetFactories()
     protected function resetRolesAndPolicyService()
```

Comment:
Maybe make this deprecated and alias it to `resetPolicyService`? Or rename right away?